lib/cmd/fh3/projects/create.js

```diff
@@ -0,0 +1,61 @@
+/* globals i18n */
+var common = require("../../../common");
+var ini = require('../../../utils/ini');
+var templates = require('../../common/templates.js');
+var util = require('util');
+
+
+module.exports = {
+  'desc' : i18n._('Create project'),
+  'examples' :
+  [{
+    cmd : 'fhc projects create --title=<title>',
```

Comment:
That's breaking change. Original command was:
`fhc projects create name` 
Title is "position parameter" to maintain backwards compatibility.

If we want to change it  then it's best to notify and update documentation as well.